README.md

```diff
@@ -196,6 +208,19 @@ conn.delete(
   /* where */ {field1: 10},
   function(err, rows, info) {
   })
+
+// Promise/async/await
+let result
+try {
+  result = await conn.insert(
+    /* table */ 'table0',
+    /* set   */ {field1: 20})
+  console.log(result[0]) // rows
```

Comment:
can you make
```
result = {
  rows: ...
  info: ...
}
```

Reply:
That's a bit hard because the return values of each function is different, so it might need to edit the original code of each function.
It can be fixed in next issue.

Reply:
interface should not be change once it's made. you can either make a pull request to thenify-all or choose another package. If you want to change thenify-all, you can also write a temporary function before the pull request is accepted.

Reply:
That's a good idea, ACCEPTED~